packages/hoppscotch-common/src/components/environments/ImportExport.vue

```diff
@@ -344,8 +344,12 @@ const handleImportToStore = async (
 ) => {
   // if there's a global env, add them to the store
   if (globalEnv) {
-    globalEnv.variables.forEach(({ key, value }) => {
-      addGlobalEnvVariable({ key, value })
+    globalEnv.variables.map((env) => {
+      if (env.secret) {
+        addGlobalEnvVariable({ key: env.key, secret: true })
+      } else {
+        addGlobalEnvVariable({ key: env.key, value: env.value, secret: false })
+      }
     })
   }
```

Comment:
```ts
  if (globalEnv) {
    globalEnv.variables.map(({ key, value, secret }) => {
      addGlobalEnvVariable({ key, value, secret })
    })
  }
```

Since we're setting `secret: false` while parsing environment variables based on Insomnia export format, we can set the incoming values straightaway specifying the type as `globalEnv?: NonSecretEnvironment`.